content/en/data-playground/dc-metro.md

```diff
@@ -23,7 +23,7 @@ weight: 5
 ---
 {{% data-playground-wrapper %}}
 
-Sample data returned by `Subscriber`:
+<a href="https://github.com/rotationalio/data-playground/tree/main/wmata" class="text-[#1D65A6] font-bold underline">Click here</a> for the code to get data such as:
```

Comment:
I think that adding another link here is a good idea since adding the diagrams would cause users to scroll up to the `Get Started` button.